tests/test_cli.py

```diff
@@ -2,14 +2,22 @@
 import tempfile
 from unittest.mock import patch
 
+import pytest
+
 from markdown_it.cli import parse
 
 
 def test_parse():
     with tempfile.TemporaryDirectory() as tempdir:
         path = pathlib.Path(tempdir).joinpath("test.md")
         path.write_text("a b c")
-        assert parse.main([str(path)])
+        assert parse.main([str(path)]) == 0
+
+
+def test_parse_fail():
+    with pytest.raises(SystemExit) as exc_info:
+        parse.main(["/tmp/nonexistant_path/for_cli_test.md"])
+    assert exc_info.value.code == 1
```

Comment:
One way to fix the test failure is:
```suggestion
    assert isinstance(exc_info.value.code, str)
```
---
Another way, that I personally would prefer but don't know if @chrisjsewell would, is go to `markdown_it.cli.parse.convert_file` and replace
```python
sys.exit('Cannot open file "{}".'.format(filename))
```
with
```python
sys.stderr.write('Cannot open file "{}".\n'.format(filename))
sys.exit(1)
```